src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -87,5 +82,41 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// collection.
         /// </summary>
         public int LeasesCollectionThroughput { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Defines a prefix to be used within a Leases collection for this Trigger. Useful when sharing the same Lease collection among multiple Triggers
+        /// </summary>
+        public string LeaseCollectionPrefix { get; set; }
```

Comment:
Do we default this to anything? I wonder if it'd be possible to use the Function name as a default prefix.

Reply:
I was thinking the same, but if I do, it would break any currently working Function because it would invalidate the current existing leases.